OpenTSDB/deployment/install-opentsdb.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash          
+
+user=$1
+password=$2
+tsd_listen_port=${3:-4242}
+proxy_domain_suffix=$4
+num_edge_nodes=${5:-1}
+
+opentsdb_tar_file=OPENTSDB.tar.gz
+opentsdb_tar_file_uri=https://hdiconfigactions.blob.core.windows.net/linuxhueconfigactionv01/$opentsdb_tar_file
```

Comment:
THis container is only for HUE related scripts:
https://hdiconfigactions.blob.core.windows.net/linuxhueconfigactionv01

Can you please put a tarball in the same folder with your installation scripts on the github?